default.nix

```diff
@@ -86,12 +86,28 @@ stdenv.mkDerivation rec {
 
   prefixKey = "-prefix ";
 
+  # Note that Coq's Makefile.dune will force sequential setup, more
+  # fine control can be gained by Nix by using dune directly on
+  # coq-core and coq-stdlib.
   enableParallelBuilding = true;
 
-  buildFlags = [ "world" ] ++ optional buildDoc "doc-html";
+  buildFlags = [ "world" ];
 
-  installTargets =
-    [ "install" ] ++ optional buildDoc "install-doc-html";
+  # TODO, building of documentation package when not in dev mode
+  # https://github.com/coq/coq/issues/16198
+  # buildFlags = [ "world" ] ++ optional buildDoc "refman-html";
```

Comment:
Open an issue

Reply:
It is already open, as noted in the comment